.drone.env

```
@@ -1,3 +1,3 @@
 # The version of OCIS to use in pipelines that test against OCIS
-OCIS_COMMITID=7b8c12079d4946c778aa43ed6262788bbe0144bc
+OCIS_COMMITID=59afd4cc665023abaf91ca5ff1715bb2e199a3b0
 OCIS_BRANCH=master
```

ocis stable-5 will use web stable-8. So maybe we should use stable-5.0 here as well
```suggestion
OCIS_COMMITID=59afd4cc665023abaf91ca5ff1715bb2e199a3b0
OCIS_BRANCH=stable-5.0
```